src/components/Stepper/index.js

```diff
@@ -0,0 +1,85 @@
+import React, { useState, useEffect } from "react";
+import Button from "../Button";
+import Icon from "../Icon";
+
+const Stepper = ({
+  steps,
+  isDisableFinish = false,
+  onFinish = () => {},
+  onStepChange = () => {},
+  finishBtnText = "Finish",
+}) => {
+  const [currentStep, setCurrentStep] = useState(0);
+
+  useEffect(() => {
+    const initialStep = steps.findIndex((step) => step.isInitial);
+    setCurrentStep(initialStep !== -1 ? initialStep : 0);
+  }, [steps]);
+
+  const handleNext = () => {
+    if (currentStep < steps.length - 1) {
+      const nextStep = currentStep + 1;
+      setCurrentStep(nextStep);
+      onStepChange(steps[nextStep]);
+    }
+  };
+
+  const handleBack = () => {
+    if (currentStep > 0) {
+      const prevStep = currentStep - 1;
+      setCurrentStep(prevStep);
+      onStepChange(steps[prevStep]);
+    }
+  };
+
+  const isLastStep = currentStep === steps.length - 1;
+
+  return (
+    <div className="Stepper-container">
```

Comment:
The class name should probably be in lowercase?
```suggestion
    <div className="stepper-container">
```